bridgetown-core/test/test_configure_command.rb

```diff
@@ -52,5 +54,49 @@ def configurations_path
 
       File.delete("#{configurations_path}/roar.rb")
     end
+
+    should "perform configuration when option provided" do
+      output = capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option:github"])
+      end
+
+      frontend_styles_path = File.expand_path(root_dir("frontend", "styles"), __dir__)
+
+      assert_match %r!create.*syntax-test.css!, output
+      assert File.exist?("#{frontend_styles_path}/syntax-test.css")
+
+      File.delete("#{frontend_styles_path}/syntax-test.css")
+    end
+
+    should "perform multiple configurations when option provided" do
+      capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option:github test_automation"])
+      end
+
+      frontend_styles_path = File.expand_path(root_dir("frontend", "styles"), __dir__)
+      assert File.exist?("#{frontend_styles_path}/syntax-test.css")
+      # TODO: This test doesn't work, but manual testing seems to prove it does work fine
+      # I _think_ it's something to do with capture_stdout not capturing everything... maybe?
+      # assert_match %r!fixture.*?Works\!!, output
+      # assert_match %r!create.*syntax-test.css!, output
```

Comment:
Couldn't get this to work?

Reply:
When invoking the command in this way, every individual argument should be a separate element in the array. So invoking it as:

@cmd.invoke(:perform_configurations, %w[test_configuration_with_option:github test_automation])

should do the trick!